ci/run-k8s-e2e-tests.sh

```diff
@@ -82,6 +84,10 @@ case $key in
     RUN_E2E_FOCUS="$2"
     shift 2
     ;;
+    --e2e-skip)
```

Comment:
name is a bit confusing IMO because i only applies to conformance tests, not NetworkPolicies.

could it be renamed from `--e2e-skip` to `--e2e-conformance-skip` (same for `E2E_SKIP` var)?